### PR TITLE
D2IQ-78969: Readd helm cli tools

### PR DESCRIPTION
**Pull request by mstruebing**
## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-78969

## Description of changes being made

I copied the pages from konvoy 1.8 to 2.0, 2.1 and 2.2

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4144.s3-website-us-west-2.amazonaws.com/dkp/kommander/2.1/cli-tools/helm/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/konvoy/2.2/cli-tools/helm/index.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+- A running Kubernetes cluster
+
+## Install Helm from the binaries
+
+Every Helm [release][helm-release] provides binaries for a variety of operating systems. These binary versions can be manually downloaded and installed.
+
+1.  Download and unpack the [latest][helm-release-latest] release. Here is an example for the version [3.3.4][helm-3.3.4].
+
+    ```bash
+    wget -c https://get.helm.sh/helm-v3.3.4-linux-arm64.tar.gz -O - | tar -xz
```
Comment (cneth) [inline pages/dkp/konvoy/2.2/cli-tools/helm/index.md:39]: See comments above about arm64 vs amd64

File: pages/dkp/konvoy/2.0/cli-tools/helm/index.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    ```bash
+    helm install mydb stable/mysql
+    ```
+
+1.  List all installed charts and check their status. Enter the following command:
+
+    ```bash
+    $ helm ls
+    NAME    NAMESPACE    REVISION    UPDATED                                 STATUS      CHART          APP VERSION
+    mydb    default      1           2020-10-06 18:06:24.41517 +0200 CEST    deployed    mysql-1.6.7    5.7.30
```
Comment (cneth) [inline pages/dkp/konvoy/2.0/cli-tools/helm/index.md:89]:
```suggestion
    mydb    default      1           2022-02-06 18:06:24.41517 +0200 CEST    deployed    mysql-8.8.25    8.0.28
```

File: pages/dkp/konvoy/2.1/cli-tools/helm/index.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+You need certain software configurations and settings before you start this procedure. This procedure requires the following items and configurations:
+
+- A running Kubernetes cluster
+
+## Install Helm from the binaries
+
+Every Helm [release][helm-release] provides binaries for a variety of operating systems. These binary versions can be manually downloaded and installed.
+
+1.  Download and unpack the [latest][helm-release-latest] release. Here is an example for the version [3.3.4][helm-3.3.4].
```
Comment (cneth) [inline pages/dkp/konvoy/2.1/cli-tools/helm/index.md:36]: Needs the arm64->amd64 changes listed above

File: pages/dkp/konvoy/2.2/cli-tools/helm/index.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+## Install Helm from a Package Manager
+
+The Helm community also contributes and maintains packages to several package managers. These are not officially supported, but are generally up-to-date. Here is how to install Helm using [Homebrew (MacOS)][homebrew-helm-mac], [Chocolatey (Windows)][homebrew-helm-windows], and [Apt (Debian/Ubuntu)][homebrew-helm-ubuntu].
+
+## Add a Helm Chart Repository
+
+After installing a Helm binary, add a Helm chart repository. To add and access the official Helm chart repository, enter the following command:
+
+```bash
+helm repo add stable https://mesosphere.github.io/charts/stable/
```
Comment (cneth) [inline pages/dkp/konvoy/2.2/cli-tools/helm/index.md:63]: Update example as above

File: pages/dkp/konvoy/2.1/cli-tools/helm/index.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+## Install Helm from a Package Manager
+
+The Helm community also contributes and maintains packages to several package managers. These are not officially supported, but are generally up-to-date. Here is how to install Helm using [Homebrew (MacOS)][homebrew-helm-mac], [Chocolatey (Windows)][homebrew-helm-windows], and [Apt (Debian/Ubuntu)][homebrew-helm-ubuntu].
+
+## Add a Helm Chart Repository
+
+After installing a Helm binary, add a Helm chart repository. To add and access the official Helm chart repository, enter the following command:
+
+```bash
+helm repo add stable https://mesosphere.github.io/charts/stable/
```
Comment (cneth) [inline pages/dkp/konvoy/2.1/cli-tools/helm/index.md:63]: Update example as above

**Review by cneth — CHANGES_REQUESTED** [review]

File: pages/dkp/konvoy/2.0/cli-tools/helm/index.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+excerpt: Helm CLI Usage
+beta: false
+enterprise: false
+---
+
+This section describes how to install the Helm CLI. Helm can be installed either from source or from pre-built binary releases. This topic focuses on installing the latest Helm binaries, which is known as Helm 3.
+
+You may encounter some situations where you need to use an older version of Helm, known as Helm 2. If so, see the Helm documentation.
+
+## What Is Helm
```
Comment (cmastr) [inline pages/dkp/konvoy/2.0/cli-tools/helm/index.md:15]: No need for this. let's just add a link to their docs.

File: pages/dkp/konvoy/2.0/cli-tools/helm/index.md [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+- A running Kubernetes cluster
+
+## Install Helm from the binaries
+
+Every Helm [release][helm-release] provides binaries for a variety of operating systems. These binary versions can be manually downloaded and installed.
+
+1.  Download and unpack the [latest][helm-release-latest] release. Here is an example for the version [3.3.4][helm-3.3.4].
+
+    ```bash
+    wget -c https://get.helm.sh/helm-v3.3.4-linux-arm64.tar.gz -O - | tar -xz
```
[the remark this answers is not in the document]
Comment (mstruebing) [reply pages/dkp/konvoy/2.0/cli-tools/helm/index.md:39]: I don't know. This depends on the system the user is using. But I'm fine with your suggestion 👍

File: pages/dkp/konvoy/2.1/cli-tools/helm/index.md (hunk 3 above)
Comment (mstruebing) [reply pages/dkp/konvoy/2.1/cli-tools/helm/index.md]: I copied over the changes from the 2.0 page to 2.1.

File: pages/dkp/konvoy/2.1/cli-tools/helm/index.md (hunk 5 above)
Comment (mstruebing) [reply pages/dkp/konvoy/2.1/cli-tools/helm/index.md]: I copied over the changes from the 2.0 page to 2.1.

File: pages/dkp/konvoy/2.0/cli-tools/helm/index.md (hunk 6 above)
Comment (mstruebing) [reply pages/dkp/konvoy/2.0/cli-tools/helm/index.md:15]: Do you mean like changing this section to something like:

```
## What Is helm

Have a look at their documentation [link]
```

Or do you mean this page as a whole?

Comment (cmastr) [reply pages/dkp/konvoy/2.0/cli-tools/helm/index.md:15]: The first option. Let's rely on Helm to describe what they do.

**Comment by msdolbey** [comment]
FYI @mstruebing - I fixed some links and so you'll need to pull if you want to make any changes.

**Comment by msdolbey** [comment]
I don't want to throw that much of a wrench in this PR - but should this be all moved over to Kommander? 
FYI @cmastr - @karishmagadiya ... hmm, maybe @gracedo @dkoshkin ?

Just thinking aloud - we'll be using helm more or less as a tool on top of DKP/provisioning, much like the rest of Kommander, so it seems like the Kommander docs might be the best place for them for now?

**Review by msdolbey — APPROVED** [review]
👍 

I made some link changes and some small nits, but overall looks good to me.

at the helm

**Comment by cmastr** [comment]
> I don't want to throw that much of a wrench in this PR - but should this be all moved over to Kommander? FYI @cmastr - @karishmagadiya ... hmm, maybe @gracedo @dkoshkin ?
> 
> Just thinking aloud - we'll be using helm more or less as a tool on top of DKP/provisioning, much like the rest of Kommander, so it seems like the Kommander docs might be the best place for them for now?

This is probably true! sigh sorry Max!

**Comment by mstruebing** [comment]
No worries! 
I moved these pages to the `kommander` releases (2.0, 2.1, 2.2) and rephrased the `what is helm` section.
Let me know if there are any more changes I should make.

**Pull request merged**